spec/virtual_total_spec.rb

```diff
@@ -265,6 +291,14 @@ def model_with_children(count)
       expect(model_with_children(2).v_total_named_books).to eq(2)
     end
 
+    it "falls back to default when virtual association is not written correctly in ruby" do
+      a = model_with_children(0)
+      # note: this is broken return, but some of our associations return nil instead of [] / none
+      expect(a).to receive(:named_books).and_return(nil)
```

Comment:
Hmm, this (and the note above it) confuses me, but I think it is because I am trying to do this review really quick (and failing).  I assume it is okay, just leaving a note for myself really.

Reply:
we have a `try` in our code that handles a `nil` coming back from an association (virtual I presume).

FWIW/ I don't like it.
struggling keeping this in here.